apps/solid-example/src/routes/__root.tsx

```
@@ -0,0 +1,36 @@
+import styleCss from "../styles.css?url"
+
+export const Route = createRootRouteWithContext()({
+  head: () => ({
+    links: [{ rel: "stylesheet", href: styleCss }],
+  }),
+  shellComponent: RootComponent,
+})
+
+function RootComponent() {
+  return (
+    <html>
```

Biome (2.3.13)

[error] Provide a lang attribute when using the html element.

Setting a lang attribute on HTML document elements configures the languageused by screen readers when no user default is specified.

(lint/a11y/useHtmlLang)